### DISCUSSION TITLE
How to deploy an image on hub.docker.com?

### DISCUSSION THREAD

**Discussion by IRediTOTO** [coollabsio/coolify#2055] [Service Template Requests]
Hi, I am trying to use the Blocks in service, but now it changed to LowCoder `https://hub.docker.com/r/lowcoderorg/lowcoder-ce`. I want to deploy Coolify to use lastest features, but I don't really know how to do that. Can you show me how please? Thank you for any help

**Comment by owldesign** [comment]
Better yet, can we add or replace Openblocks with now new lowcoder? Openblocks has been abandoned and replaced with lowcoder.
